docs/src/chapters/prezentacja.adoc

```diff
@@ -1 +1,287 @@
+na stronie głównej, następuje przekierowania do pierwszego ekranu procesu, czyli formularza płatności. W górnej
+części ekranu wydać pasek informujący o aktualnym ekranie procesu oraz komunikat o braku możliwości późniejszego
+wyświetlenia tokenów po zakończeniu płatności. W centralnej części ekranu znajduje się sam formularz. Od niezalogowanego
+użytkownika wymagane jest podanie liczby tokenów oraz adresu email. Służy on do wygenerowania specjalnego tokena JWT,
+pozwalającego na dokonanie płatności oraz jej identyfikację w bazie danych. W przyszłych wydaniach planowane jest
+również dodanie serwisu wysyłającego zakupione produkty na wskazany adres email. Formularz zawiera również informacje
+o oferowanym produkcie, czyli opis oraz cenę. Po wysłaniu formularza użytkownik zostaje przekierowany na drugi
+ekran procesu.
+
+.Formularz płatności dla niezalogowanego użytkownika.
+image::../images/system_presentation/webapp/anonymous_qb_1.png[]
+
+Drugi ekran służy do opłacenia zamówienia. Zawiera on informację o finalnej cenie zakupu, a także komponent
+wyświetlający dwa kody QR. Pierwszy zawiera żądanie zapłaty w sieci Lightning, a drugi adres węzła. Pomiędzy kodami
+QR użytkownik może przełączać za pomocą przełącznika w dolnej części komponentu. Oba kody można zeskanować przy pomocy
+portfela Bitcoin wspierającego płatności w sieci Lightning. Na ekranie znajduje się również minutnik, pokazujący
+czas pozostały na wykonanie płatności. Po jego upływie transakcja automatycznie zmieni status na anulowany. Gdy
+użytkownik opłaci żądanie zapłaty, zostanie automatycznie przekierowany na następny ekran.
+
+.Ekran z kodem QR dla niezalogowanego użytkownika.
+image::../images/system_presentation/webapp/anonymous_qb_2.png[]
+
+Ostatnim ekranem jest podsumowanie płatności. Zawiera on listę zakupionych tokenów. Można je wyświetlić bądź skopiować
+do schowka przy pomocy odpowiednich przycisków. Przy kopiowaniu, a także wielu innych akcjach w aplikacji internetowej
+wyświetlane są w lewym dolnym roku powiadomienia o dokonanej akcji. Przycisk _close_ powoduje powrót na pierwszy
+formularza płatności. Aby wrócić na stronę główną, należy kliknąć nagłówek strony w lewym górnym rogu ekranu.
+
+.Podsumowanie płatności dla niezalogowanego użytkownika.
+image::../images/system_presentation/webapp/anonymous_qb_3.png[]
+
+Aby zyskać dostęp do indywidualnej historii płatności, należy założyć konto w serwisie. Formularz znajduje się
+pod przyciskiem _register_ na stronie głównej aplikacji. Formularz ten zawiera tylko cztery pola. Są to
+adres email, imię i nazwisko, oraz dwa pola dotyczące wprowadzenia hasła.
+
+.Formularz rejestracji użytkownika.
+image::../images/system_presentation/webapp/register.png[]
+
+Wszystkie pola w całej aplikacji zawierają walidację danych wejściowych. W tym przypadku sprawdzany jest między innymi
+odpowiedni format adresu email, czy też wymogi długości i znaków specjalnych hasła. Użytkownik jest proszony o
```

Comment:
"czy też wymogi długości i znaków specjalnych hasła" to trzeba inaczej napisać, albo może następne zdanie z tego stworzyć

Reply:
zmieniłem na "wymogi dotyczące siły hasła"